src/prompt-converters.js

```diff
@@ -267,6 +268,16 @@ function convertGooglePrompt(messages, model) {
     const isMultimodal = visionSupportedModels.includes(model);
     let hasImage = false;
 
+    let sys_prompt = '';
```

Comment:
> Will that merge examples too?

it will, the behavior shouldn't be any different from the other converter minus the userName + charName prepend to examples if they're passed through as parameters. i thought about copying from the other two to include that little change, but seeing as the cohere implementation doesn't make use of the userName/charName params when making the request i didn't see much of a point.